packages/button/src/Button.tsx

```diff
@@ -17,12 +17,18 @@ export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    * The variant to use. Defaults to "primary".
    */
   variant?: 'primary' | 'secondary' | 'ghost' | 'overlay' | 'critical'
+
+  /**
+   * Whether the button is interactive or not. Defaults to false.
+   */
+  disabled?: boolean
```

Comment:
Since this is a native prop on button (disabled by default), worth adding it here?

Reply:
Ahh right yes, i can remove as a prop.  To be able to toggle in `Storybook` I have to leave the JSDOC though, correct?

Reply:
I wonder if you can create a story with disabled set in `Template.args`

Reply:
I think I will leave this for the next round of button updates.

Reply:
Let's not wait for next round since we should do it right from the get-go